torch/_torch_docs.py

```diff
@@ -9640,12 +9640,14 @@ def merge_dicts(*dicts):
     tensor([ 0.1815, -0.8917, -0.3031])
 """)
 
-add_docstr(torch.slogdet,
-           r"""
+add_docstr(torch.slogdet, r"""
 slogdet(input) -> (Tensor, Tensor)
 
 Calculates the sign and log absolute value of the determinant(s) of a square matrix or batches of square matrices.
 
+.. note:: :func:`torch.slogdet` is deprecated. Please use :func:`torch.linalg.slogdet`
```

Comment:
"instead, ..." -> "instead."

torch.slogdet is actually just an alias for torch.linalg.slogdet now, so both are similar to NumPy's numpy.linalg.slogdet. We just want people to use the new name.